tango/integrations/wandb/torch_train_callback.py

```diff
@@ -15,36 +18,47 @@ class WandbTrainCallback(TrainCallback):
     the :class:`~tango.integrations.torch.TorchTrainStep` that logs training and
     validation metrics to W&B.
 
+    This can be used with any :class:`~tango.workspace.Workspace` implementation,
+    including :class:`WandbWorkspace`.
```

Comment:
All the customization you put in here, do we also need that for the lightning integration?

Reply:
I haven't tried to use PTL's `WandbLogger`, but looking at [its source code](https://pytorch-lightning.readthedocs.io/en/stable/_modules/pytorch_lightning/loggers/wandb.html#WandbLogger) it seems like it would work okay with the `WandbWorkspace` because it reuses any existing W&B run when there is one.